src/twisted/python/test/test_release.py

```diff
@@ -738,17 +738,23 @@ def test_build(self):
         """
         Creates and builds a fake Sphinx project using a L{SphinxBuilder}.
         """
-        self.createFakeSphinxProject()
-        self.builder.build(self.sphinxDir)
-        self.verifyBuilt()
+        try:
+            self.createFakeSphinxProject()
+            self.builder.build(self.sphinxDir)
+            self.verifyBuilt()
+        except CalledProcessError as e:
+            raise Exception(f"{e.stdout} {e.stderr}") from e
```

Comment:
the failure was:

```python
Traceback (most recent call last):
  File "/home/runner/work/twisted/twisted/.tox/alldeps-withcov-posix/bin/sphinx-build", line 5, in <module>
    from sphinx.cmd.build import main
  File "/home/runner/work/twisted/twisted/.tox/alldeps-withcov-posix/lib/python3.10/site-packages/sphinx/cmd/build.py", line 25, in <module>
    from sphinx.application import Sphinx
  File "/home/runner/work/twisted/twisted/.tox/alldeps-withcov-posix/lib/python3.10/site-packages/sphinx/application.py", line 32, in <module>
    from sphinx.config import Config
  File "/home/runner/work/twisted/twisted/.tox/alldeps-withcov-posix/lib/python3.10/site-packages/sphinx/config.py", line 23, in <module>
    from sphinx.util import logging
  File "/home/runner/work/twisted/twisted/.tox/alldeps-withcov-posix/lib/python3.10/site-packages/sphinx/util/__init__.py", line 35, in <module>
    from sphinx.util import smartypants  # noqa
  File "/home/runner/work/twisted/twisted/.tox/alldeps-withcov-posix/lib/python3.10/site-packages/sphinx/util/smartypants.py", line 33, in <module>
    from sphinx.util.docutils import __version_info__ as docutils_version
  File "/home/runner/work/twisted/twisted/.tox/alldeps-withcov-posix/lib/python3.10/site-packages/sphinx/util/docutils.py", line 31, in <module>
    from sphinx.util.typing import RoleFunction
  File "/home/runner/work/twisted/twisted/.tox/alldeps-withcov-posix/lib/python3.10/site-packages/sphinx/util/typing.py", line 34, in <module>
    from types import Union as types_Union
ImportError: cannot import name 'Union' from 'types' (/opt/hostedtoolcache/Python/3.10.0-rc.1/x64/lib/python3.10/types.py)
```

Reply:
so that's here: https://github.com/sphinx-doc/sphinx/pull/9513/files#diff-f41e19a9b77f4670d4dd1ea61f80d518c93a4403112fd7e91253dac47cf55fe5L37